generation/internal/python.go

```diff
@@ -7,122 +7,100 @@ import (
 
 const pythonOrb = "circleci/python@2"
 
-func defaultSteps(l labels.Label, hasManagePy bool) []config.Step {
-	steps := []config.Step{
-		{
-			Type:    config.OrbCommand,
-			Command: "python/install-packages",
-		},
+func testSteps(ls labels.LabelSet) []config.Step {
+	hasManagePy := ls[labels.FileManagePy].Valid
+	hasSetupPy := ls[labels.FileSetupPy].Valid
+	hasTox := ls[labels.FileToxIni].Valid
+	hasPipenv := ls[labels.PackageManagerPipenv].Valid
+	hasPoetry := ls[labels.PackageManagerPoetry].Valid
```

Comment:
~ Instead of having different flags for each of the different package managers I wonder if we should have a single `PackageManager` label that is set to the intended value (pipenv, poetry, ....etc)

Reply:
It gets tricky since we are dealing with multiple package managers (pipenv, poetry, setuptools) and different test runners (manage, pytest, tox). Keeping the labels separate helps avoid needing labels within labels. 

I've updated the generation logic to hide some of the details in interface implementations - let me know if that helps.